### DISCUSSION TITLE
https://cucoders.dev/publicaciones/aprezcuba24/nextjs-typeorm-postgresql/

### DISCUSSION THREAD

**Discussion by giscus** [CuCodersCommunity/cucoderscommunity.github.io#140] [Page Comments]
# https://cucoders.dev/publicaciones/aprezcuba24/nextjs-typeorm-postgresql/

Nextjs ha llegado para revolucionar el desarrollo web. La adopción de los Server Component ha supuesto un cambio radical, que es hasta cierto punto es un regreso a los inicios del desarrollo web, donde el servidor interviene también en la generación de interfaces de usuario.
También pienso que los perfiles de programadores, frontend y backend, que por mucho tiempo se han manejados, con esta nueva filosofía dejan un poco de ser útiles. Y los desarrolladores fullstack tendrán ventaja.

https://cucoders.dev/publicaciones/aprezcuba24/nextjs-typeorm-postgresql/

**Comment by pedroomar23** [comment]
Buen articulo 💪
